airbyte-integrations/connectors/source-mysql-v2/src/test/test/resources/expected-spec.json

```diff
@@ -0,0 +1,340 @@
+{
```

Comment:
this file was copied twice by mistake, this one is the bad one which needs to be deleted